src/main/java/eu/europeana/metis/sandbox/controller/DatasetController.java

```diff
@@ -118,23 +101,43 @@ public DatasetIdDto harvestDatasetFromFile(
   public DatasetIdDto harvestDatasetFromURL(
       @ApiParam(value = "name of the dataset", required = true) @PathVariable(value = "name") String datasetName,
       @ApiParam(value = "country of the dataset", required = true, defaultValue = "Netherlands") @RequestParam Country country,
-      @ApiParam(value = "language of the dataset", required = true, defaultValue = "nl") @RequestParam Language language,
+      @ApiParam(value = "language of the dataset", required = true, defaultValue = "Dutch") @RequestParam Language language,
       @ApiParam(value = "dataset records URL to download in a zip file", required = true) @RequestParam String url) {
     checkArgument(namePattern.matcher(datasetName).matches(),
         "dataset name can only include letters, numbers, _ or - characters");
-    List<ByteArrayInputStream> records = harvestService.harvest(url);
+    List<ByteArrayInputStream> records = harvestService.harvestZipUrl(url);
 
     checkArgument(records.size() < maxRecords,
         "Amount of records can not be more than " + maxRecords);
     var datasetObject = datasetService.createDataset(datasetName, country, language, records);
     return new DatasetIdDto(datasetObject);
   }
 
-  /**
-   * GET API call to return the progress status of a given dataset id
-   * @param datasetId The given dataset id to look for
-   * @return The report of the dataset status
-   */
```

Comment:
In the meantime, add here as well a javadoc for `harvestDatasetOaiPmh`

Reply:
My bad. It was the resolve conflicts. Didn't notice it. I'll fix it